File: app/src/main/java/com/victoria/bleled/util/arch/AbsentLiveData.java
package com.victoria.bleled.util.arch;

import androidx.lifecycle.LiveData;

public class AbsentLiveData<T> extends LiveData<T> {
    AbsentLiveData() {
        postValue(null);
    }

    public static <T> AbsentLiveData<T> create() {
        return new AbsentLiveData<T>();
    }
}

File: app/src/main/java/com/victoria/bleled/base/internal/AbsentLiveData.kt
package com.victoria.bleled.base.internal

import androidx.lifecycle.LiveData

class AbsentLiveData<T> internal constructor() : LiveData<T>() {
    init {
        postValue(null)
    }

    companion object {
        @JvmStatic
        fun <T> create(): AbsentLiveData<T> {
            return AbsentLiveData()
        }
    }
}